packages/cli/src/init.ts

```diff
@@ -435,6 +436,75 @@ To start the server, run the following command:
 Then, try look up an actor from your server:
 
   ${colors.bold(colors.green("fedify lookup http://localhost:3000/users/john"))}
+`,
+    }),
+  },
+  next: {
+    label: "Next.js",
+    runtimes: ["node"],
+    init: (_, __, packageManager) => ({
+      label: "Next.js",
+      runtimes: ["node"],
+      command: [
+        "npx",
```

Comment:
Shouldn't this vary depending on `packageManager`?


```suggestion
        ...(packageManager === "npm" ? "npx" : [packageManager, "dlx"]),
```

Reply:
Changed at 26d28e9a62583920790b9fdf031bb0aa111ece31!
I writed the code with `create next-app` for pnpm and yarn, but it wasn't working. As you said, the answer was `dlx`! Always thanks to you.